mlos_bench/mlos_bench/tests/tunables/tunable_to_configspace_test.py

```diff
@@ -101,6 +103,9 @@ def configuration_space() -> ConfigurationSpace:
             TunableValueKind.RANGE,
         )
     )
+    hp = spaces["kernel_sched_latency_ns"]
+    assert isinstance(hp, NumericalHyperparameter)
+    _monkey_patch_quantization(hp, quantization_bins=10)
```

Comment:
Can we remove the `# TODO` above this?  Or is that waiting on #813 ?

Reply:
yep, we can remove that comment now. the unit tests are here - although we probably should add a few more to #813 
I'll take a look once we merge in this one